### DISCUSSION TITLE
Is it possible to patch legacy USB 1.1 automatically?

### DISCUSSION THREAD

**Discussion by mikezang** [dortania/OpenCore-Legacy-Patcher#1152] [Reports being investigated]
Though I installed Montery and Sonoma, then updateed Sequoia with the same USBHub, wired keyboard and mouse, I can't use those wired keyboard and mouse, I also tried several other USB HUBs, I still can't use my keyboard and mouse!
I have suggestion, is it possible to patch USB 1.1 automatically without click button "OK"? so that we can use wireless keyboard and mouse....
I installed Sequoia on External SSD, now I only can use it.
I updated to Sonoma 14.7.2 to Internal SSD, now I can't patch USB 1.1! I can only try reboot and try again, sometimes the wired keyboard and mouse maybe work by chance, but I am not sure when the next chance will come...
![IMG_7273](https://github.com/user-attachments/assets/eb6a8d82-8bf9-4231-91d5-928b21218e9e)
Internal SSD Sonoma 14.7.1
![Sonoma14 7 1](https://github.com/user-attachments/assets/a1dd8f02-0346-46ce-b4db-d88c99fba849)
External SSD Sequoia 15.1
![Sequoia15 1](https://github.com/user-attachments/assets/83eb16c3-49c1-4192-bc13-5e9c009ec0a8)

**Comment by JonnyTech** [comment]
Duplicate #1151
